pyproject.toml

```diff
@@ -48,8 +48,8 @@ jupyter = [
   'trame-vuetify>=2.3.1',
   'trame>=2.5.2',
 ]
-# Pinned versions of core dependencies
-pinned = [
+
```

Comment:
new-lines were added between some `project.optional-dependencies` to _avoid_ sorting these keys